docs/admin/configuration/authentication-and-user-management/index.md

```diff
@@ -15,3 +15,29 @@ OpenCloud employs a dual-track authentication strategy:
    - Keycloak as the recommended OIDC provider for larger installations
    - Support for Azure AD, EntraID, ADFS through Keycloak
    - Enterprise-focused solution
+
+## Production Authentication with Keycloak and LDAP
+
+For production environments, we recommend using Keycloak with LDAP integration. This setup provides a robust authentication system that can scale to enterprise needs.
+
+OpenCloud now supports two authentication modes when using Keycloak with LDAP:
+
+### Shared User Directory Mode
+
+In this mode, LDAP serves as a central user directory for both Keycloak and OpenCloud.
+
+- LDAP is the source of truth for user information
+- Both Keycloak and OpenCloud read user data directly from LDAP
+- User accounts must exist in LDAP before they can log in
+- Manual user provisioning required in LDAP
+
+### Autoprovisioning Mode
+
+In this mode, Keycloak automatically creates and updates users in LDAP upon first login.
```

Comment:
When reading this, I immediately want to know things like:
- how does OpenCloud/Keycloak actually get write access to the LDAP server? (I suppose something has to be done in the LDAP server to grant "write/admin" permission to OpenCloud/Keycloak so it can create new users)
- how is user-creation controlled? Can "anyone on the big bad internet" sign-up to the OpenCloud/Keycloak login interface? For example, as a new staff member of a company, can I just "autoprovision" myself???